src/cv/insert_frame.py

```diff
@@ -16,7 +18,36 @@
 def insert_frame(camera_id, stream_id, frame_id, frame, objs):
     """ Inserts frame into cv_frames table """
     stmt = session.prepare("""
-        INSERT INTO streams.cv_frames (camera_id, stream_id, frame_id, frame, objects_detected)
+        INSERT INTO streams.cv_frames
+        (camera_id, stream_id, frame_id, frame, objects_detected)
         VALUES(?, ?, ?, ?, ?)
     """)
     stmt_results = session.execute(stmt, [camera_id, stream_id, frame_id, frame, objs])
+
+def store_training_data(class_name):
+    """ Inserts images into database """
+    training_data_paths = list(paths.list_images(settings.TRAINING_DATA))
+    for img_path in training_data_paths:
+        with open(img_path, 'rb') as file_data:
+            stmt = session.prepare("""
+                INSERT INTO streams.training_data (data_id, class_name, insert_date, data)
+                VALUES(?, ?, ?, ?)
+            """)
+            stmt_results = session.execute(stmt,
+                [uuid.uuid4(), class_name, time.time(), file_data.read()])
+    os.system("rm -rf " + settings.TRAINING_DATA)
+
+def retrieve_training_data():
+    """ Obtains the training data from database """
+    if not os.path.exists(settings.DATASET):
+        os.system('mkdir ' + settings.DATASET)
+    res = session.execute("SELECT class_name, data from streams.training_data")
```

Comment:
Now that you have the timestamp itself, your query should also include that part itself to serve as a way of filtering the data you don't want. Otherwise, there wasn't a point in including the timestamp. You should probably save a timestamp somewhere that has the value of the last time retrieved so you can retrieve from timestamps that are greater than that the next time.

Reply:
yes